eng/pipelines/jobs/archetype-sdk-client.yml

```diff
@@ -0,0 +1,81 @@
+parameters:
```

Comment:
For consistency can you put these under a templates directory? i.e. `eng/pipelines/templates/jobs`

Reply:
Legit thought about this before saying

> nah it's fine 

🤦‍♂ update incoming.